scripts/install_plugins.py

```diff
@@ -66,9 +66,9 @@
   os.chdir(cwd)
   subsOut = os.popen('git config --file .gitmodules --name-only --get-regexp path').read()
   ## subsInit are the initialized ones
-  subsInit = [x.split(' ')[1] for x in os.popen('git submodule status').read().split(os.linesep) if x.strip() != '']
+  subsInit = [x.split(' ')[1] for x in os.popen('git submodule status').read().split("\n") if x.strip() != '']
   submods = []
-  for m, sub in enumerate(subsOut.split(os.linesep)):
+  for m, sub in enumerate(subsOut.split("\n")):
```

Comment:
Does it depend on operation systems?

Reply:
I checked online, it looks good to me.

Reply:
yes...
okay

Reply:
we should not ever use os.linesep in case of "text" mode...because for win... we have "/n/r" (/r" too)